### PR TITLE
install versioneer

### PR DESCRIPTION
**Pull request by avirshup**
Part of #13

### CODE REVIEW

File: geometric/__init__.py [hunk 1]
```diff
@@ -5,4 +5,8 @@
 
 from . import molecule
 from . import optimize
-from . import engine
\ No newline at end of file
+from . import engine
+
+from ._version import get_versions
+__version__ = get_versions()['version']
```
Comment (dgasmith) [inline geometric/__init__.py]: Can you add a `__git_revision__ = versions['full-revisionid']`?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[versioneer]
+VCS = git
```
Comment (dgasmith) [inline setup.cfg]: Can you add an explicit `style = pep440`?

**Comment by avirshup** [comment]
Done. LMK if you'd like it squashed

**Comment by leeping** [comment]
Reading the documentation of versioneer now. Is it standard that we include versioneer.py in the root folder?

**Comment by avirshup** [comment]
It's standard, yeah - you _might_ be able to put it elsewhere with a little bit of `sys.path` hacking? I've never tried it though.

**Comment by leeping** [comment]
Thanks - merging. :)

**Pull request merged**